### ISSUE TITLE
[Issue] Poor performance early in the day

### ISSUE THREAD

**Issue by jesseorosz** [iexg/IEX-API#909]
**Clearly describe the problem**
Querying the company API for 8900 symbols depends on the day varies greatly. Anywhere between one minute and seven minutes. Happens with all APIs

**What time did the problem occur? (with timezone)**
10AM EST

**What is the expected behavior?**
Takes one minute

**Your environment (e.g. browser, OS, API version, language):**
win10

**API URL (if applicable)**

**What are the steps to reproduce the bug?**
Basically when you need the data the most (during the market hours) it's an order of magnitude slower than off market hours

**Additional context**
Provide some sort of query performance status page
Allow a single call to return all possible records - rather than 9000 calls to company, one single call that returns 9000 records.

**Comment by evancarter-iex** [comment]
@jesseorosz Is this currently an issue?

**Comment by jesseorosz** [comment]
Yes, except I am getting service disconnects now.

company and chart both disconnects after 500-1000 calls

i'm having no issues with quote

**Comment by evancarter-iex** [comment]
@jesseorosz What exactly is returned?

**Comment by jesseorosz** [comment]
calling it through python:

aiohttp.client_exceptions.ServerDisconnectedError: None

**Comment by jesseorosz** [comment]
I'm also only making one call at a time, running async

**Comment by evancarter-iex** [comment]
Hmm what ip are you hitting us from?

**Comment by evancarter-iex** [comment]
Closing this, please file another issue. Thanks
